instrumentation/micrometer/micrometer-1.5/library/src/main/java/io/opentelemetry/instrumentation/micrometer/v1_5/OpenTelemetryFunctionTimer.java

```diff
@@ -44,7 +44,7 @@
         otelMeter
             .counterBuilder(name + ".count")
             .setDescription(Bridging.description(id))
-            .setUnit("1")
+            .setUnit("{invocations}")
```

Comment:
Shouldn't that be singular?
```suggestion
            .setUnit("{invocation}")
```